docker-compose-aws-logging-letsencrypt.yml

```diff
@@ -7,8 +7,8 @@ services:
     logging:
       driver: awslogs
       options:
-        awslogs-region: eu-west-1
-        awslogs-group: canarytokens-dev
+        awslogs-region: ${CANARYTOKENS_LOGGING_REGION:-eu-west-1}
```

Comment:
My first thought was that we may want to use the standard `CANARY_` prefix, but as this is not being included in the settings objects I suspect it's better as it is.

Reply:
hmmm its a good thought to raise. I hadn't considered it. I'll leave for now.